libsolidity/formal/SMTChecker.cpp

```diff
@@ -571,6 +577,22 @@ void SMTChecker::endVisit(Identifier const& _identifier)
 	}
 }
 
+void SMTChecker::visitTypeConversion(FunctionCall const& _funCall)
+{
+	solAssert(_funCall.annotation().kind == FunctionCallKind::TypeConversion, "");
+	if (_funCall.arguments().size() > 1)
+		m_errorReporter.warning(
+			_funCall.location(),
+			"Assertion checker does not support type conversion with multiple argumets."
+		);
+	else
+	{
+		defineExpr(_funCall, expr(*_funCall.arguments().at(0)));
+		// Restrict the range in case of cast to smaller type.
```

Comment:
What about int to bytes conversion? Do all conversion retain the numerical value? Probably yes...

Reply:
Yes. But I just noticed that this code is not enough.